EasyPost.Tests/ServicesTests/CarrierAccountServiceTest.cs

```diff
@@ -43,6 +44,26 @@ public async Task TestCreate()
             Assert.StartsWith("ca_", carrierAccount.Id);
         }
 
+        [Fact]
+        [CrudOperations.Create]
+        [Testing.Parameters]
+        public async Task TestCreateWithAlternateEndpoint()
+        {
+            UseVCR("create_with_alternate_endpoint");
+
+            // FedEx and UPS should hit a different creation endpoint
+            try
+            {
+                CarrierAccount carrierAccount = await CreateSpecificCarrierAccount("FedexAccount");
+            }
+            catch (UnknownApiError e)
```

Comment:
Needs indenting?

Reply:
Not sure what you're referring to

Reply:
Oh, this was showing up weird for me. Double checked and it's fine.